plugins/outputs/groundwork/groundwork_test.go

```diff
@@ -9,10 +9,13 @@ import (
 	"testing"
 
 	"github.com/gwos/tcg/sdk/clients"
+	sdkLog "github.com/gwos/tcg/sdk/log"
 	"github.com/gwos/tcg/sdk/transit"
 	"github.com/stretchr/testify/require"
 
 	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/logger"
+	logAdapter "github.com/influxdata/telegraf/plugins/outputs/groundwork/slog"
```

Comment:
Please don't add a sub-package but simply add the log-adapter to this package as a separate file e.g. named `log_adapter.go`.

Reply:
Done